<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> [Twisted-Python] could plugin cache get currupted?
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:twisted-python%40twistedmatrix.com?Subject=Re%3A%20%5BTwisted-Python%5D%20could%20plugin%20cache%20get%20currupted%3F&In-Reply-To=%3C47EBCB93.2080007%40evotex.ch%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=utf-8">
   <LINK REL="Previous"  HREF="049709.html">
   <LINK REL="Next"  HREF="049713.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[Twisted-Python] could plugin cache get currupted?</H1>
    <B>Gabriel Rossetti</B> 
    <A HREF="mailto:twisted-python%40twistedmatrix.com?Subject=Re%3A%20%5BTwisted-Python%5D%20could%20plugin%20cache%20get%20currupted%3F&In-Reply-To=%3C47EBCB93.2080007%40evotex.ch%3E"
       TITLE="[Twisted-Python] could plugin cache get currupted?">mailing_lists at evotex.ch
       </A><BR>
    <I>Thu Mar 27 10:30:11 MDT 2008</I>
    <P><UL>
        <LI>Previous message (by thread): <A HREF="049709.html">[Twisted-Python] Re: [ANN] Twisted 8.0
</A></li>
        <LI>Next message (by thread): <A HREF="049713.html">[Twisted-Python] could plugin cache get currupted?
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#49711">[ date ]</a>
              <a href="thread.html#49711">[ thread ]</a>
              <a href="subject.html#49711">[ subject ]</a>
              <a href="author.html#49711">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>Hello,

I coded a program that uses Twisted's plugin system. I have noticed that 
if I interrupt the application using Ctrl+C (SIGINT) that when I run it 
next time I get this :

  File &quot;/usr/lib/python2.5/site-packages/twisted/plugin.py&quot;, line 227, 
in getPlugins
    allDropins = getCache(package)
--- &lt;exception caught here&gt; ---
  File &quot;/usr/lib/python2.5/site-packages/twisted/plugin.py&quot;, line 178, 
in getCache
    provider = namedAny(module.__name__ + '.' + moduleName)
  File &quot;/usr/lib/python2.5/site-packages/twisted/python/reflect.py&quot;, 
line 375, in namedAny
    obj = getattr(obj, n)
exceptions.AttributeError: 'module' object has no attribute 'SendFile'

Could this mean that the plugin cache is corrupt? If so, How can I keep 
it from getting corrupted (is there a safe way of closing it)?

Thanks,
Gabriel


</PRE>

<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message (by thread): <A HREF="049709.html">[Twisted-Python] Re: [ANN] Twisted 8.0
</A></li>
	<LI>Next message (by thread): <A HREF="049713.html">[Twisted-Python] could plugin cache get currupted?
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#49711">[ date ]</a>
              <a href="thread.html#49711">[ thread ]</a>
              <a href="subject.html#49711">[ subject ]</a>
              <a href="author.html#49711">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="https://twistedmatrix.com/cgi-bin/mailman/listinfo/twisted-python">More information about the Twisted-Python
mailing list</a><br>
</body></html>
